sycl/include/sycl/info/ext_intel_device_traits.def

```diff
@@ -21,6 +21,7 @@ __SYCL_PARAM_TRAITS_SPEC(ext::intel, device, current_clock_throttle_reasons, std
 __SYCL_PARAM_TRAITS_SPEC(ext::intel, device, fan_speed, int32_t, UR_DEVICE_INFO_FAN_SPEED)
 __SYCL_PARAM_TRAITS_SPEC(ext::intel, device, min_power_limit, int32_t, UR_DEVICE_INFO_MIN_POWER_LIMIT)
 __SYCL_PARAM_TRAITS_SPEC(ext::intel, device, max_power_limit, int32_t, UR_DEVICE_INFO_MAX_POWER_LIMIT)
+__SYCL_PARAM_TRAITS_SPEC(ext::intel, device, luid, detail::luid_type, UR_DEVICE_INFO_LUID)
```

Comment:
Why do we need both this and the one above?

Reply:
Ah sorry, both are not required, only the part in `ext_intel_device_traits.def` is.

The version in `device_traits.def` I believe is meant to support the old style of requesting device traits of `ext_intel_device_info_uuid` which has been deprecated.